File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_deptpay.cs
// EXPECTED: 5847.95
using System;
public static class DeptpayRef {
  public static void Main() {
    double total = 111111.11;
    int n = 19;
    double avg = Math.Floor(total / n * 100.0) / 100.0;
    Console.WriteLine(avg.ToString("0.00"));
  }
}

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_deptpay.java
// EXPECTED: 5847.95
public final class DeptpayRef {
  public static void main(String[] args) {
    double total = 111111.11;
    int n = 19;
    double avg = Math.floor(total / n * 100.0) / 100.0;
    System.out.printf("%.2f%n", avg);
  }
}
